src/main/java/com/amazonaws/services/dynamodbv2/AcquireLockOptions.java

```diff
@@ -120,6 +139,20 @@ public AcquireLockOptionsBuilder withDeleteLockOnRelease(final Boolean deleteLoc
             return this;
         }
 
+        /**
+         * Sets whether or not to allow acquiring locks if the lock does not exist already
+         *
+         * @param acquireOnlyIfLockAlreadyExists If true, locks will not be granted on items which do not already
+         *                                       exist.
+         *                                       If false (default value), lock item will be created if it does not
+         *                                       exist already.
+         * @return a reference to this builder for fluent method chaining
+         */
+        public AcquireLockOptionsBuilder withAcquireOnlyIfLockAlreadyExists(final Boolean acquireOnlyIfLockAlreadyExists) {
```

Comment:
discuss the difference between acquireOnlyIfLockAlreadyExists and updateExistingLockRecord AcquireLockOptions

Reply:
added to PR summary